addons/l10n_lu/data/template/account.tax-lu.csv

```diff
@@ -1,1007 +1,1007 @@
 "","","","","","","","","","","tax","refund","-702","","lu_2020_account_461411"
-"lu_2015_tax_VP-PA-16","480","","16%","16-S-S","16.0","percent","sale","tax_group_16","","base","invoice","+901","",""
+"lu_2015_tax_VP-PA-16","480","","16%","16% S","16.0","percent","sale","tax_group_16","","base","invoice","+901","",""
 "","","","","","","","","","","tax","invoice","+902","","lu_2020_account_461411"
 "","","","","","","","","","","base","refund","-901","",""
 "","","","","","","","","","","tax","refund","-902","","lu_2020_account_461411"
-"lu_2015_tax_VP-PA-13","481","","13%","13-S-S","13.0","percent","sale","tax_group_13","","base","invoice","+903","",""
+"lu_2015_tax_VP-PA-13","481","","13%","13% S","13.0","percent","sale","tax_group_13","","base","invoice","+903","",""
 "","","","","","","","","","","tax","invoice","+904","","lu_2020_account_461411"
 "","","","","","","","","","","base","refund","-903","",""
 "","","","","","","","","","","tax","refund","-904","","lu_2020_account_461411"
-"lu_2015_tax_VP-PA-7","482","","7%","7-S-S","7.0","percent","sale","tax_group_7","","base","invoice","+905","",""
+"lu_2015_tax_VP-PA-7","482","","7%","7% S","7.0","percent","sale","tax_group_7","","base","invoice","+905","",""
 "","","","","","","","","","","tax","invoice","+906","","lu_2020_account_461411"
 "","","","","","","","","","","base","refund","-905","",""
 "","","","","","","","","","","tax","refund","-906","","lu_2020_account_461411"
-"lu_2011_tax_VP-PA-3","503","","3%","3-S-S","3.0","percent","sale","tax_group_3","","base","invoice","+031","",""
+"lu_2011_tax_VP-PA-3","503","","3%","3% S","3.0","percent","sale","tax_group_3","","base","invoice","+031","",""
 "","","","","","","","","","","tax","invoice","+040","","lu_2020_account_461411"
 "","","","","","","","","","","base","refund","-031","",""
 "","","","","","","","","","","tax","refund","-040","","lu_2020_account_461411"
-"lu_2015_tax_VP-PA-8","505","","8%","8-S-S","8.0","percent","sale","tax_group_8","","base","invoice","+705","",""
+"lu_2015_tax_VP-PA-8","505","","8%","8% S","8.0","percent","sale","tax_group_8","","base","invoice","+705","",""
 "","","","","","","","","","","tax","invoice","+706","","lu_2020_account_461411"
 "","","","","","","","","","","base","refund","-705","",""
 "","","","","","","","","","","tax","refund","-706","","lu_2020_account_461411"
-"lu_2015_tax_SANS","506","","0%","0-P-Tax-Free","0.0","percent","purchase","tax_group_0","","base","invoice","","",""
+"lu_2015_tax_SANS","506",Tax free,"0%",0%,"0.0","percent","purchase","tax_group_0","","base","invoice","","",""
 "","","","","","","","","","","tax","invoice","","",""
 "","","","","","","","","","","base","refund","","",""
 "","","","","","","","","","","tax","refund","","",""
-"lu_2015_tax_SANS_sale","507","","0%","0-S-Tax-Free","0.0","percent","sale","tax_group_0","","base","invoice","","",""
+"lu_2015_tax_SANS_sale","507","","0%",0% S S,"0.0","percent","sale","tax_group_0","","base","invoice","","",""

```

Comment:
```suggestion
"lu_2015_tax_SANS_sale","507","","0%","0% S S","0.0","percent","sale","tax_group_0","","base","invoice","","",""

```
:scream:

Reply:
I did'nt had the regex for this loca, so the others should be good :rofl: